src/locales/en.json

```diff
@@ -29,6 +29,7 @@
 "inline": "Inline",
 "label.about": "About",
 "label.about.app": "About CloudStack",
+"label.accept": "Accept",
```

Comment:
where is this used @davidjumani ?

Reply:
https://github.com/apache/cloudstack-primate/blob/master/src/views/offering/AddNetworkOffering.vue#L108